client/src/game/shapes/interfaces.ts

```diff
@@ -4,6 +4,9 @@ export interface Tracker {
     name: string;
     value: number;
     maxvalue: number;
+    draw: boolean;
+    primary_color: string;
+    secondary_color: string;
```

Comment:
camelCase these!

In the typescript world we do everything with camelCase except those things that arrive from python which are snake_case, which is why the Server* interfaces have snake_case